src/main/resources/data.sql

```diff
@@ -1,11 +1,7 @@
-DROP TABLE IF EXISTS product;
+INSERT INTO product (id, name, description) VALUES
+(1, 'product1', 'product1 description'),
+(2, 'product2', 'product2 description');
 
-CREATE TABLE product (
-                              id INT AUTO_INCREMENT  PRIMARY KEY,
-                              name VARCHAR(50) NOT NULL,
-                              description VARCHAR(250) NOT NULL
-);
-
-INSERT INTO product (name, description) VALUES
-('product1', 'product1 description'),
-('product2', 'product2 description');
\ No newline at end of file
+INSERT INTO user (email, password, name) VALUES
+('email1@email.com', 'pass1', 'name1'),
+('email2@email.com', 'pass2', 'name2');
```

Comment:
Passwords are hashed during login, plaintext passwords will not be recognized:

```suggestion
('email1@email.com', '$2a$10$H27RX92/KnQywQOF7l/VCeIa2g77.4pIbldPwrr.Ar8Qw.JXPGokC', 'name1'),
('email2@email.com', '$2a$10$Lhdy33w8QaV/bJutecweGOrSOAyjUOxz6IAexCtOyRl9ghfsXeWRW', 'name2');
```

> 2021-01-16 13:49:30.670  WARN 7006 --- [nio-8080-exec-6] o.s.s.c.bcrypt.BCryptPasswordEncoder     : Encoded password does not look like BCrypt

Reply:
you're a life saver!